pyproject.toml

```diff
@@ -55,6 +55,7 @@ dependencies = [
     "wget",
     "ai21",
     "cohere",
+    "docker",
```

Comment:
@StochasticRomanAgeev remove this. Let us not do any change to xTuring source code right now